### PR TITLE
Update DDF version to 2.25.2

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Update DDF version to 2.25.2
NOTE: WebHDFS adapter has been disabled for now

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@cjlange 
@aaronilovici 
#### How should this be tested? (List steps with links to updated documentation)
Install and verify basic DDF replication capability on a DDF 2.25.2 based system.
#### What are the relevant tickets?
Fixes #XXXX

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: distributions/osgi/features/replication/src/main/feature/feature.xml [hunk 1]
```diff
@@ -40,6 +40,6 @@
         <bundle>mvn:replication/replication-ui/${project.version}</bundle>
         <bundle>mvn:replication/commons/${project.version}</bundle>
         <bundle>mvn:replication-adapters/ddf-adapter/${project.version}</bundle>
-        <bundle>mvn:replication-adapters/webhdfs-adapter/${project.version}</bundle>
+<!--        <bundle>mvn:replication-adapters/webhdfs-adapter/${project.version}</bundle>-->
```
Comment (aaronilovici) [inline distributions/osgi/features/replication/src/main/feature/feature.xml]: Should this be removed? Or maybe leave a comment about why it's commented out (if HDFS is not supported here)?

**Review by aaronilovici — APPROVED** [review]

File: distributions/osgi/features/replication/src/main/feature/feature.xml (hunk 1 above)
Comment (cjlange) [inline distributions/osgi/features/replication/src/main/feature/feature.xml]: A comment here could be useful for anyone looking at this in the future.

Comment (clockard) [reply distributions/osgi/features/replication/src/main/feature/feature.xml]: removed commented out line

Comment (clockard) [reply distributions/osgi/features/replication/src/main/feature/feature.xml]: removed commented out line

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Review by cjlange — APPROVED** [review]

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**